Sources/Ink/Internal/Readable.swift

```diff
@@ -10,10 +10,10 @@ internal protocol Readable {
 
 extension Readable {
     static func readOrRewind(using reader: inout Reader) throws -> Self {
-        guard reader.previousCharacter != "\\" else {
-            throw Reader.Error()
-        }
-
+//        guard reader.previousCharacter != "\\" else {
+//            throw Reader.Error()
+//        }
+// Not sure why this is here it is interfering with proper backslash handling
```

Comment:
To enable top-level fragments, like lists and headings to be escaped.

Reply:
Well the best argument is that the escaping behaviour change is quite subtle and breaking to make later rather than earlier.  The history of all the handling of escapes is in the evolution of CommonMark. I am not a zealot here; but I respect John MacFarlane, author of pandoc, as he built the ultimate markdown rosetta tool and knows all the divergences.

I know is makes the code less pretty. I am thinking of it as a swift challenge test to match one of the premier libraries Cmark. These little items could make the onboarding experience of all other markdown files nasty.

The only other way to solve it is to use pandoc as a preprocessor to change CommonMark to Ink so you would have to brush up on haskell :)

I am just pounding out these backslash tests and have not done lists etc. yet.